tests/providers/dbt/cloud/hooks/test_dbt.py

```diff
@@ -394,6 +394,70 @@ def test_trigger_job_run_with_additional_run_configs(
         )
         hook._paginate.assert_not_called()
 
+    @pytest.mark.parametrize(
+        argnames="conn_id, account_id",
+        argvalues=[(ACCOUNT_ID_CONN, None), (NO_ACCOUNT_ID_CONN, ACCOUNT_ID)],
```

Comment:
nitpick: Why are `argnames`, `argvalues` needed here but not the first decorator? Let's keep them consistent